src/metabase/automagic_dashboards/rules.clj

```diff
@@ -285,28 +277,14 @@
   [rule]
   (transduce (map (comp count ancestors)) + (:applies_to rule)))
 
-(defn- load-rule
-  [^Path f]
-  (try
-    (let [entity-type (file->entity-type f)]
-      (-> f
-          .toUri
-          slurp
-          yaml/parse-string
-          (assoc :rule        entity-type
-                 :specificity 0)
-          (update :applies_to #(or % entity-type))
-          rules-validator
-          (as-> rule (assoc rule :specificity (specificity rule)))))
-    (catch Exception e
-      (log/error (trs "Error parsing {0}:\n{1}"
-                      (.getFileName f)
-                      (or (some-> e
-                                  ex-data
-                                  (select-keys [:error :value])
-                                  u/pprint-to-str)
-                          e)))
-      nil)))
+(defn- make-rule
```

Comment:
what is a "rule" in this context?

Reply:
It's the old x-ray nomenclature. In the new version these become specs as well